src/AccessControl/owner.py

```diff
@@ -264,6 +264,8 @@ def ownerInfo(user, getattr=getattr):
     if uid is None:
         return uid
     db = aq_parent(aq_inner(user))
+    if not db:
```

Comment:
I know this PR is already closed, but this check should perhaps be:

    if db is None:

As it is, if the container is a typical Zope folder, it could trigger a `__len__()` call, which would be needlessly inneficient here.

Reply:
@leorochael Thank you for looking into this PR. You are right.
I created #93 include the fix you suggested.